BUILDING.txt

```diff
@@ -0,0 +1,74 @@
+Build instructions for Apache Commons Crypto
+
+----------------------------------------------------------------------------------
+Requirements:
+
+* Unix System
+* JDK 1.6 or above (environment variable JAVA_HOME must be set)
+* Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy
+  Files (if running unit tests)
+* Maven 3.0 or above
+* Make
+* Openssl devel 1.0.1c or above 
+
+----------------------------------------------------------------------------------
+Install JCE Unlimited Strength Jurisdiction Policy Files:
+
+Download the Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files from Oracle:
+
+For JDK 1.6: http://www.oracle.com/technetwork/java/javase/downloads/jce-6-download-429243.html
+For JDK 1.7: http://www.oracle.com/technetwork/java/javase/downloads/jce-7-download-432124.html
+For JDK 1.8: http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html
+
+Install JCE Unlimited Strength Jurisdiction Policy Files to JDK:
+
+Copy downloaded local_policy.jar and US_export_policy.jar to <java-home>/jre/lib/security/
+
+----------------------------------------------------------------------------------
+Upgrade Openssl version in Mac:
```

Comment:
Do we need name this section to Verify OpenSSL version?
My idea is:
check OpenSSL version is common for all OS.

upgrade OpenSSL version is different for different OS, we can have a small section as:
Upgrade Openssl in Mac:
...

For Linux systems, there are a lot of distributions, we can simply mention:
Upgrade OpenSSL in Linux:
You can follow your OS distribution instructions to upgrade OpenSSL to a proper version.